.github/workflows/go.yml

```diff
@@ -40,16 +42,39 @@ jobs:
         run: ./test_all.sh
 
   quick-tests:
+    strategy:
+      matrix:
+        containerGoVer:
+          - "1.20"
+          - "1.21-rc"
     runs-on: ubuntu-latest
-    container: golang:1.20
+    container: golang:${{ matrix.containerGoVer }}
     steps:
     - uses: actions/checkout@v3
       with:
         persist-credentials: false
 
-
     - name: Go Build
       run: go build -v ./...
 
     - name: Unit Tests
       run: go test -v ./...
+
+  gvc:
```

Comment:
I'd like to write this out as `govulncheck`. It shows up as the name of the test phase in the Commits tab, so it's good to have a meaningful name. I think we chose the short name `b` over in Boulder because we wanted to make more room to display the matrix parameters.